src/containers/WalletsPage/Deposit.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
```

Comment:
@AntonBurduzha Here is the way I was thinking:

Components:
- HistoryTable
- Deposit
- Withdraw
- WalletLayout (Tabs for select Deposit / Withdraw but it's optional and we can put it in container)

Container:
- WalletPage

WalletPage can have router, and assemble together
```
<WalletLayout>
  <Deposit /> or <Withdraw />
  <History />
</WalletLayout>
```
or put the tabs directly in WalletPage

@AntonBurduzha 
Don't create a 1 to 1 correspondence with Deposit component and Deposit Containers
The container is the WalletPage and contains {SideBar, Deposit, Withdraw, History} components

TODO @AntonBurduzha :
- [ ] Remove Deposit container
- [ ] Remove Withdraw container
- [ ] Remove SideBar container
- [ ] connect() all states at the WalletPage level